.eslintrc.json

```diff
@@ -175,5 +175,22 @@
     "no-unmodified-loop-condition": "off",
     "accessor-pairs": "off",
     "@typescript-eslint/no-namespace": "off"
-  }
+  },
+  "overrides": [
+    // Disabled rules only for test files
+    {
+      "files": ["**/*.spec.ts"],
+      "rules": {
+        "no-unused-expressions": "off",
```

Comment:
There's a eslint plugin for mocha that could be worth looking into, as to not disable some rules https://www.npmjs.com/package/eslint-plugin-mocha

Reply:
I think we are more interested in this plugin: https://www.npmjs.com/package/eslint-plugin-chai-friendly